specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -2334,6 +2352,33 @@
         }
       }
     },
+    "BackendPool": {
+      "type": "object",
+      "description": "Backend pool information",
+      "properties": {
+        "services": {
+          "type": "array",
+          "description": "The list of backend entities belonging to a pool.",
+          "items": {
+            "$ref": "#/definitions/BackendPoolItem"
+          },
+          "minItems": 1
+        }
+      }
+    },
+    "BackendPoolItem": {
+      "type": "object",
+      "description": "Pool service information",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The unique ARM id of the backend entity. The ARM id should refer to an already existing backend entity.",
+          "example":{
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/backends/proxybackend"
+          }
```

Comment:
Is this resource coming in the PUT payload ? Do you plan to add a linked access check to make sure that the calling user has access to this resource